streams/src/test/java/org/apache/kafka/streams/StreamsConfigTest.java

```diff
@@ -62,4 +65,23 @@ public void testGetRestoreConsumerConfigs() throws Exception {
         assertEquals(returnedProps.get(ConsumerConfig.CLIENT_ID_CONFIG), "client-restore-consumer");
         assertNull(returnedProps.get(ConsumerConfig.GROUP_ID_CONFIG));
     }
+
+    @Test
+    public void testSerde() {
```

Comment:
The `test` prefix was inherited from Scala code and I agree we do not need it. @miguno Could you file a JIRA to remove this prefix across the code base?